lib/enode.ts

```diff
@@ -16,19 +16,35 @@ export async function clearUsers(userIDs: string[]) {
   }
 }
 
-// Not all properties
 export type Inverter = {
   id: string;
   vendor: string;
   chargingLocationId: string | null;
   lastSeen: string;
   isReachable: boolean;
-  information: {};
-  location: {};
   productionState: {
-    productionRate: number;
+    productionRate: number | null;
+    isProducing: boolean | null;
+    totalLifetimeProduction: number | null;
+    lastUpdated: string | null;
+  };
+  information: {
+    id: string;
+    brand: string;
+    model: string;
+    siteName: string;
+    installationDate: string;
+  };
+  location: {
+    longitute: number | null;
+    latitude: number | null;
   };
 };
+
+export type Inverters = {
+  inverters: Inverter[];
+};
```

Comment:
Reading through the docs, it seems like this type (at least if it's being used as the return type for the inverters endpoint) should just be equal to `Inverter[]`—there shouldn't be a key called `inverters`

Reply:
That makes sense! In the endpoint, I think we return Inverters(inverters=inverters), where inverters is a key. Would this need to change in the endpoint, or would it still work if I just used Inverter[] as the type in the inverterFetcher?

Reply:
ok, sounds good, you're correct then!